opentelemetry-api/src/opentelemetry/metrics/_internal/instrument.py

```diff
@@ -33,8 +33,8 @@
 )
 
 # pylint: disable=unused-import; needed for typing and sphinx
-from opentelemetry import _metrics as metrics
-from opentelemetry._metrics._internal.observation import Observation
+from opentelemetry import metrics
+from opentelemetry.metrics._internal.observation import Observation
```

Comment:
```suggestion
from opentelemetry.metrics import Observation
```

Reply:
I think it is ok to import from internal API components inside another API component. We do the same in the SDK, and that is needed to avoid circular import errors.